.github/workflows/stability-js-ci.yml

```diff
@@ -0,0 +1,110 @@
+name: Stability - Keycloak JavaScript CI
+
+on:
+  workflow_dispatch:
+
+env:
+  MAVEN_ARGS: "-B -nsu -Daether.connector.http.connectionMaxTtl=25"
+  RETRY_COUNT: 0
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-keycloak:
+    name: Build Keycloak
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build Keycloak
+        uses: ./.github/actions/build-keycloak
+
+      - name: Prepare archive for upload
+        run: |
+          mv ./quarkus/dist/target/keycloak-999.0.0-SNAPSHOT.tar.gz ./keycloak-999.0.0-SNAPSHOT.tar.gz
+
+      - name: Upload Keycloak dist
+        uses: actions/upload-artifact@v4
+        with:
+          name: keycloak
+          path: keycloak-999.0.0-SNAPSHOT.tar.gz
+
+  generate-test-seed:
+    name: Generate Test Seed
+    runs-on: ubuntu-latest
+    outputs:
+      seed: ${{ steps.generate-random-number.outputs.value }}
+    steps:
+      - name: Generate random number
+        id: generate-random-number
+        shell: bash
+        run: |
+          echo "value=$(shuf -i 1-100 -n 1)" >> $GITHUB_OUTPUT
+
+  admin-ui-e2e:
```

Comment:
We should also have a workflow/job for the account console.

Reply:
Since the UI tests run quickly I just added both admin and account E2E to the same workflow